api/app/signals/apps/api/serializers/signal.py

```diff
@@ -565,15 +564,17 @@ class Meta:
         )
 
 
-class SignalGeoSerializer(GeoFeatureModelSerializer):
-    # For use with the "geography" action
-    location = GeometryField(source='location.geometrie')
+class SignalGeoSerializer(serializers.Serializer):
+    feature = serializers.JSONField()
 
     class Meta:
-        model = Signal
-        id_field = False
-        geo_field = 'location'
-        fields = ['id', 'created_at']
+        fields = ('feature', )  # feature must annotated on the queryset
+        list_serializer_class = GeoFeatureModelListSerializer  # We still use the GeoFeatureModelListSerializer
+
+    def to_representation(self, instance):
+        # TODO: Make this nicer instead of this hacky solution :)
```

Comment:
Agree, or small comment with explanation of what is going on (stripping one property out of the representation that happens to contain all we need).

Reply:
This is still a P.O.C., so will make this code much nicer if we agree on using this solution.